en/docs/reference/customize-product/customizations/adding-a-user-signup-workflow-using-bps.md

```diff
@@ -98,7 +98,7 @@ This section explains how to attach a custom workflow to the user signup operati
 #### Configuring the Business Process Server
 
 !!! note
-    Follow this sub section, only if you will be using the **Business Process Server(BPS)** for the business process tasks. If not please refer the sub section for [Configuring the Entreprise Integrator.](/learn/consume-api/customizations/adding-a-user-signup-workflow/#configuring-the-enterprise-integrator)
+    Follow this sub section, only if you will be using the **Business Process Server(BPS)** for the business process tasks. If not please refer the sub section for [Configuring the Enterprise Integrator.](/learn/consume-api/customizations/adding-a-user-signup-workflow/#configuring-the-enterprise-integrator)
```

Comment:
This is not part of this exercise, but this link will appear in a page not found as there is no {{base_path}} in front of the path.

Reply:
Fixed.